autoinstrumentation/python/requirements.txt

```diff
@@ -1,53 +1,53 @@
-opentelemetry-distro==0.36b0
+opentelemetry-distro==0.37b0
 # We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
 # injected auto-instrumentation, where it has a strict dependency on the OS / Python version the artifact is built for.
-opentelemetry-exporter-otlp-proto-http==1.15.0
+opentelemetry-exporter-otlp-proto-http==1.16.0
 
-opentelemetry-propagator-b3==1.15.0
-opentelemetry-propagator-jaeger==1.15.0
+opentelemetry-propagator-b3==1.16.0
+opentelemetry-propagator-jaeger==1.16.0
 opentelemetry-propagator-aws-xray==1.0.1
 
-opentelemetry-instrumentation==0.36b0
-opentelemetry-propagator-ot-trace==0.36b0
+opentelemetry-instrumentation==0.37b0
+opentelemetry-propagator-ot-trace==0.37b0
 
 # Copied in from https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation
 # except for aws-lambda
-opentelemetry-instrumentation-aio-pika==0.36b0
-opentelemetry-instrumentation-aiohttp-client==0.36b0
-opentelemetry-instrumentation-aiopg==0.36b0
-opentelemetry-instrumentation-asgi==0.36b0
-opentelemetry-instrumentation-asyncpg==0.36b0
-opentelemetry-instrumentation-boto==0.36b0
-opentelemetry-instrumentation-boto3sqs==0.36b0
-opentelemetry-instrumentation-botocore==0.36b0
-opentelemetry-instrumentation-celery==0.36b0
-opentelemetry-instrumentation-confluent-kafka==0.36b0
-opentelemetry-instrumentation-dbapi==0.36b0
-opentelemetry-instrumentation-django==0.36b0
-opentelemetry-instrumentation-elasticsearch==0.36b0
-opentelemetry-instrumentation-falcon==0.36b0
-opentelemetry-instrumentation-fastapi==0.36b0
-opentelemetry-instrumentation-flask==0.36b0
-opentelemetry-instrumentation-grpc==0.36b0
-opentelemetry-instrumentation-httpx==0.36b0
-opentelemetry-instrumentation-jinja2==0.36b0
-opentelemetry-instrumentation-kafka-python==0.36b0
-opentelemetry-instrumentation-logging==0.36b0
-opentelemetry-instrumentation-mysql==0.36b0
-opentelemetry-instrumentation-pika==0.36b0
-opentelemetry-instrumentation-psycopg2==0.36b0
-opentelemetry-instrumentation-pymemcache==0.36b0
-opentelemetry-instrumentation-pymongo==0.36b0
-opentelemetry-instrumentation-pymysql==0.36b0
-opentelemetry-instrumentation-pyramid==0.36b0
-opentelemetry-instrumentation-redis==0.36b0
-opentelemetry-instrumentation-requests==0.36b0
-opentelemetry-instrumentation-sklearn==0.36b0
-opentelemetry-instrumentation-sqlalchemy==0.36b0
-opentelemetry-instrumentation-sqlite3==0.36b0
-opentelemetry-instrumentation-starlette==0.36b0
-opentelemetry-instrumentation-tornado==0.36b0
-opentelemetry-instrumentation-tortoiseorm==0.36b0
-opentelemetry-instrumentation-urllib==0.36b0
-opentelemetry-instrumentation-urllib3==0.36b0
-opentelemetry-instrumentation-wsgi==0.36b0
+opentelemetry-instrumentation-aio-pika==0.37b0
```

Comment:
do we need to update this too?

Reply:
part of the instructions were to bump all the auto-inst to latest.

Reply:
No, as part of the release we should bump only the released auto-instrumentation agents in versions.txt